release/readme.html

```diff
@@ -205,119 +254,192 @@ <h2><a id="Weather_Data_63"></a>Weather Data</h2>
         about the included (or other) files. Weather data for more than 2100 locations in more than 100 countries can be
         downloaded on the <a href="https://energyplus.net/weather">EnergyPlus Weather Page</a>.</p>
 
-    <h2><a id="Highlights_of_this_release"></a>Highlights of this release (V9.2.0)</h2>
+    <h2>Important Changes (OK, back to this library thing...)</h2>
+    <p>There are three changes in this release of EnergyPlus which rely on different software dependencies.</p>
+    <dl>
+        <dt>C/Python API</dt>
+        <dd>
+            A significant change to this release is the ability to call EnergyPlus as a library.  Up until now, the way
+            users interacted with EnergyPlus has always been to setup an input file, and call the EnergyPlus(.exe) binary to run
+            the file, then gather results as needed from output files.  While this is still available, a new approach has been
```

Comment:
There's also the existing FMU external interface methods. Not sure if that's relevant here for what you're getting at.

Reply:
Well, technically, we also had a way for users to call E+ as a library, but it was clunky, not well tested, and not well documented.  I can add a little nod to external interface though.